ophyd/controls/area_detector.py

```diff
@@ -638,7 +638,6 @@ def configure(self, state=None):
         self._write_plugin('AutoSave', 1, self._file_plugin)
         self._write_plugin('NumCapture', 0, self._file_plugin)
         self._write_plugin('FileWriteMode', 2, self._file_plugin)
-        self._write_plugin('EnableCallbacks', 1, self._file_plugin)
```

Comment:
This bit of code has had a tremendous about of random hacking without any
tests so great skepticism is called for.

On Fri, Oct 30, 2015, 22:16 Stuart Wilkins <notifications@github.com> wrote:

> In ophyd/controls/area_detector.py
> <https://github.com/NSLS-II/ophyd/pull/163#discussion_r43566961>:
>
> > @@ -638,7 +638,6 @@ def configure(self, state=None):
> >          self._write_plugin('AutoSave', 1, self._file_plugin)
> >          self._write_plugin('NumCapture', 0, self._file_plugin)
> >          self._write_plugin('FileWriteMode', 2, self._file_plugin)
> > -        self._write_plugin('EnableCallbacks', 1, self._file_plugin)
>
> Ok 10-4.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NSLS-II/ophyd/pull/163/files#r43566961>.
>